### DISCUSSION TITLE
AI has social consequences, but who pays the price? Tech companies’ problem with ‘ethical debt’

### DISCUSSION THREAD

**Discussion by Aaron25564448** [1101-GS4719-C/Python-Community-GS4719-C#1257] [General]
https://theconversation.com/ai-has-social-consequences-but-who-pays-the-price-tech-companies-problem-with-ethical-debt-203375
隨著科技的發展，大家應該都有學過，會產生所謂的社會成本，也就是對社會帶來的負面影響。但AI發展意外的，但也不意外的產生的「道德成本」，而且更甚於過往各種科技。像是在各公司競爭下快速進化的AI，當被拿來犯罪、造成爭議時，企業是否也應該負起責任。我個人覺得還是一樣那句話，科技始終來自人心，然而確實，AI競爭造成的發展真的是有些的快，社會的體制目前很明顯是跟不上AI造成的問題，AI繪圖就是其中一個例子。就目前來看，要減少AI發展帶來的社會成本和道德成本，還是要讓社會也個適應期，讓AI穩定滑順的融入社會。

**Comment by KevinCheng42** [comment]
從過去科技蓬勃發展的歷程如手機、電腦，他們都因為過於快速發展造成社會的法律或觀念無法同時跟進，產生社會成本。我們應該從歷史得到教訓，此次AI正要興起，立法者應該要盡速立下相關法條，而我們也要用正向的觀念看待AI技術，不要透過此技術為非作歹！

**Comment by curtis1017** [comment]
社會成本常常是企業家忽略的事情，但是社會成本常常會造成社會有極大的傷害，我覺得未來需要有更多法律來制衡企業家常常忽略的社會成本，或是將外部成本內部化，這樣才會使這個社會更美好。
